pages/common/ibmcloud-login.md

```diff
@@ -0,0 +1,28 @@
+# ibmcloud login
+
+> Log in to the IBM Cloud.
+> More information: <https://cloud.ibm.com/docs/cli?topic=cli-ibmcloud_cli#ibmcloud_login>.
+
+- Log in by using interactive prompts:
```

Comment:
Are there multiple prompts? If not I would use
```diff
- Log in by using interactive prompts:
+ Log in by using an interactive prompt:
```

Reply:
Well, there is a prompt for a username, followed by one for a password and then one for a region, so I thought that counts as multiple prompts.

Reply:
I'm not sure if this would still count as one prompt. Multiple prompts sounds pretty odd IMO.

Reply:
OK, I'll change it.